modules/enterprise-management/src/test/java/com/ibm/cloud/platform_services/enterprise_management/v1/EnterpriseManagementIT.java

```diff
@@ -1,433 +1,563 @@
-                .newRequest(Key.Type.STRING, String.class).reduce(false).keys(subscriptionEmail).includeDocs(true)
-                .build().getResponse().getRows();
-        for (ViewResponse.Row<String, String> row : result) {
-            assertNotNull(row.getDocument());
-            activationToken = row.getId();
-        }
-    }
-
-    @Test
-    public void test07_VerifyAMAccount() throws Throwable {
-        accountManagementService.verifyAMAccount(subscriptionEmail, activationToken, token, config.get("AM_HOST"));
-        Thread.sleep(35000);
-        String response2 = accountManagementService.getAccountByAccountId(subsAccountId, token, config.get("AM_HOST"));
-        JSONObject obj = new JSONObject(response2);
-        JSONObject obj2 = obj.getJSONObject("entity");
-        subscriptionId = obj2.getString("subscription_id");
-        ownerIamId = obj2.getString("owner_iam_id");
-    }
-
-    @Test
-    public void test08_PatchAccountSubscription() throws Throwable {
-        accountManagementService.patchAccountSubscription(token, subsAccountId, subscriptionId, config.get("AM_HOST"));
-    }
-
-    @Test
-    public void test09_CreateEnterpriseUsingSubscriptionAccount() throws Throwable {
-        CreateEnterpriseOptions createEnterpriseOptions = new CreateEnterpriseOptions.Builder()
-                .name("IBM-" + new Date().getTime()).sourceAccountId(subsAccountId).primaryContactIamId(ownerIamId)
-                .domain("IBM-" + new Date().getTime() + ".com").build();
-        Response<CreateEnterpriseResponse> response = service.createEnterprise(createEnterpriseOptions).execute();
-        CreateEnterpriseResponse responseObj = response.getResult();
-        assertNotNull(responseObj);
-        enterpriseId = responseObj.getEnterpriseId();
-        enterpriseAccountId = responseObj.getEnterpriseAccountId();
-    }
-
-    @Test
-    public void test10_GetAccountById() throws Throwable {
-        String response2 = accountManagementService.getAccountByAccountId(subsAccountId, token, config.get("AM_HOST"));
-        JSONObject obj = new JSONObject(response2);
-        JSONObject obj2 = obj.getJSONObject("entity");
-        parent = obj2.getString("parent");
-    }
-
-    @Test
-    public void test11_GetAccountByIdUsingEnterprise() throws Throwable {
-        Thread.sleep(35000);
-        GetAccountOptions getAccountByIdOptions = new GetAccountOptions.Builder().accountId(enterpriseAccountId)
-                .build();
-        Response<Account> response = service.getAccount(getAccountByIdOptions).execute();
+  public EnterpriseManagement service = null;
+  public static Map<String, String> config = null;
+  final HashMap<String, InputStream> mockStreamMap = TestUtilities.createMockStreamMap();
+  final List<FileWithMetadata> mockListFileWithMetadata = TestUtilities.creatMockListFileWithMetadata();
+
+  private Random rnd = new Random();
+
+  String authType = null;
```

Comment:
Some of these variables are no longer used since we're not using the account-mgmt apis any longer.  Please remove them.   This would be authType, authUrl, and apiKey.